### PR TITLE
Only include cgosymbolizer on Linux

### PR DESCRIPTION
**Pull request by albertvaka**
### Motivation

Windows is not supported and it seems it can trigger 100% CPU usage on MacOS Big Sur [1].

[1] https://github.com/golang/go/issues/45558

### CODE REVIEW

File: pkg/collector/python/memory_linux.go [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// +build python
+
+package python
+
+import (
+	_ "github.com/benesch/cgosymbolizer"
```
Comment (truthbk) [inline pkg/collector/python/memory_linux.go:11]: I'm fine with this, though strictly speaking I think we'd want to only avoid this import for darwin (note that there are freebsd ports of the datadog-agent).

**Review by truthbk — COMMENTED** [review]
Please make s note of https://github.com/golang/go/issues/45558 in the PR description. Do we know if this affects `amd64` _and_ `arm64` builds?

**Review by olivielpeau — APPROVED** [review]

**Comment by olivielpeau** [comment]
The issue of 100% CPU usage on MacOS Big Sur affects the `amd64` build, although we haven't confirmed yet that this is the root cause.

File: pkg/collector/python/memory_linux.go (hunk 1 above)
Comment (mx-psi) [reply pkg/collector/python/memory_linux.go:12]: [README says](https://github.com/benesch/cgosymbolizer#platform-support)
> Only Linux and macOS are supported.

**Review by mx-psi — APPROVED** [review]

File: pkg/collector/python/memory_linux.go (hunk 1 above)
Comment (truthbk) [reply pkg/collector/python/memory_linux.go:12]: Yeah, I _think_ we can actually now support other platforms too, but we weren't doing it ourselves, so this is fine.

**Pull request merged**